actsims/util.py

```diff
@@ -46,6 +47,15 @@ def get_noise_seed(self, set_idx, sim_idx, data_model, season, patch, array, pat
     def get_poisson_seed(self, set_idx, sim_idx):
         return (set_idx, 0, self.POISSON, sim_idx)
 
+    def get_tiled_noise_seed(self, set_idx, sim_idx, data_model, qid, tile_idx):
+        ret = (set_idx, 0, self.TILED_NOISE, sim_idx)
+        dm = data_model
+
+        assert(dm.name in self.dmdict.keys())
+        dm_idx = self.dmdict[dm.name]
+        qid_idx = dmint.get_all_dr5_qids().index(qid)
```

Comment:
Hm, that makes sense to me though. I don't think this would break with the new algorithm, since the "new-ness" is captured in the variable TILED_NOISE. I just thought it made sense organizationally to have `dr5` be assigned a unique value, in the absence of any other reason. But this reason seems like a good one, so I will defer to what you think is most appropriate.

That's a good point on the qid. You could hash the qid string itself, which just locks you into the string representations of the arrays (I think we are locked in anyway). The only hard part here is how to hash the string. Built-in python hash() generates integers that are too large for numpy seeds. Maybe just truncate the integer to ~8 digits? We'd probably be ok on collisions?